coralproject/talk
# Add classname hooks for viewing-options-sort-option default and active state

## What does this PR do?

This PR adds classname hooks for the `SortOption` component to enable alternate styling for it's default and active (`input[type="radio"]:checked`) states.

## How do I test this PR?

- Start Talk using `talk-plugin-viewing-options` and `talk-plugin-sort*` plugins
- In the comment stream click the Viewing Options menu and check the classnames (default and active states) are correctly applied to the `label` element
